test/fixtures/secret_key.pem

```diff
@@ -0,0 +1 @@
+pem-key-file-secret-key
```

Comment:
could we make the "secret" fixtures a little more obviously not secret, just to lower the heart rate of any of our CIO colleagues who dig into this?

`pem-key-file-not-actually-a-secret-key` would be fine. :) Same for some of the info in the `secret_key.json` file above.